data/cookbook/networking/tcp-client-lwt/00-lwt.md

```diff
@@ -0,0 +1,71 @@
+---
+packages:
+- name: lwt
+  version: "5.7.0"
+- name: logs
+  version: "0.7.0"
+sections:
+- filename: main.ml
+  language: ocaml
+  code_blocks:
+  - explanation: |
+      Defines some constants about the remote host. The `(let*)` operator permits the chaining of multiple Lwt statements.
+    code: |
+      let (let*) = Lwt.bind
+
+      let connect_host = "localhost"
+      let connect_service = "smtp"
+  - explanation: |
+      We setup the `Logs` parameters and create and let's the Lwt schedules the following instructions.
```

Comment:
This sentence needs some clarification. "We setup the Logs parameters and create the Lwt schedules using the following instructions." Is this what you mean?